docs/RESOURCES.md

```diff
@@ -0,0 +1,7 @@
+## Useful Resources
+
+* [StackOverflow](http://stackoverflow.com/questions/tagged/java).
+* [The official java subreddit](https://www.reddit.com/r/java)
```

Comment:
Is this official?

Also, please capitalize java -> Java in the link description :)